packages/react-error-overlay/webpack.config.iframe.js

```diff
@@ -23,15 +23,15 @@ module.exports = {
         oneOf: [
           // Source
           {
-            test: /\.js$/,
+            test: /\.(js)|(tsx?)$/,
```

Comment:
Just curious as to why these changes were made the error overlay is a completely separate project unrelated to building the app?

Reply:
Yeah I'm manually pulling some of the changes you've made on this branch into my code base, but didn't pull these in they don't affect anything as far as I can tell.